stack/stack-setup-2.yaml

```diff
@@ -104,101 +104,114 @@ ghc:
             content-length: 110798512
             sha1: 2ba321f45918f477a1c69d153ca793b4a11540e0
         8.0.2:
-            url: "https://github.com/commercialhaskell/ghc/releases/download/ghc-8.0.2-release/ghc-8.0.2-i386-deb7-linux.tar.xz"
+            url: "https://downloads.haskell.org/~ghc/8.0.2/ghc-8.0.2-i386-deb7-linux.tar.xz"
+            mirror_url: "https://github.com/commercialhaskell/ghc/releases/download/ghc-8.0.2-release/ghc-8.0.2-i386-deb7-linux.tar.xz"
```

Comment:
For now, can you make the all `mirror_url`s comments instead (`#mirror_url: "…"`)?  When specifying a local copy of the setup-info (`stack --resolver=ghc-X.Y.Z setup --setup-info-yaml=path/to/stackage-content/stack/stack-setup-2.yaml`), Stack will warn about unrecognized fields (`Warning: path/to/stackage-content/stack/stack-setup-2.yaml: Unrecognized field in GHCDownloadInfo: mirror_url` x 100).

Reply:
Done